modules/zio/logging/src/main/scala/tofu/logging/impl/ZUniversalContextLogging.scala

```diff
@@ -0,0 +1,22 @@
+package tofu.logging.impl
+
+import org.slf4j.{LoggerFactory, Marker}
+import tofu.logging.{LoggedValue, Logging}
+import zio._
+
+class ZUniversalContextLogging[R](name: String, fctx: (LoggedValue => Unit) => URIO[R, Unit])
```

Comment:
> fctx: (LoggedValue => Unit) => URIO[R, Unit]

We definitely need a scaladoc with a note on unsafety of this argument

Reply:
I feel we can simplify that down to 
`ctxLog: URIO[R, LoggedValue]`
and use direct flatMap in the logger

Reply:
Oh, I've got it. I'll fix

Reply:
Done